gradio/data_classes.py

```diff
@@ -27,7 +27,19 @@
 
     # Map V2 method calls to V1 implementations.
     # Ref: https://docs.pydantic.dev/latest/migration/#changes-to-pydanticbasemodel
-    class BaseModel(BaseModelV1):
+    class BaseModelMeta(type(BaseModelV1)):
+        def __new__(mcs, name, bases, dct):  # noqa: N804
```

Comment:
I have thought `mcs` is the correct conventional name, but learned that `cls` is also used in metaclass 😮 
Will use `cls` for consistency.